src/components/story-block.js

```diff
@@ -12,20 +12,23 @@ const StoryBlock = ({
   description,
   fullText,
   imageRight = false,
+  bigText = true,
 }) => (
-  <section>
-    <Container>
-      <Row>
-        <Col sm={{ span: '6', order: `${imageRight ? 'last' : 'first'} ` }}>
-          <Img fluid={imageSource} alt={altText} />
-        </Col>
-        <Col sm={6}>
-          {description}
-          <ReactMarkdown source={fullText} />
-        </Col>
-      </Row>
-    </Container>
-  </section>
-)
+    <section>
+      <Container className={bigText ? "story-block-text story-block-big-text" : "story-block-text"} >
```

Comment:
this is cool! and I see how that matches with the css classes you declared. However, whats the plan for using this prop? As is, I don't see anywhere on this PR where its actually being used. Instead its just true everywhere.

Reply:
Got them set to false by default